src/util/srp.cpp

```diff
@@ -478,6 +478,7 @@ static void calculate_H_AMK(SRP_HashAlgorithm alg, unsigned char *dest, const mp
 	hash_final(alg, &ctx, dest);
 }
 
+#ifndef WIN32
```

Comment:
This file isn't maintained here. Make a pull request to https://github.com/est31/csrp-gmp instead.